src/solvers/refinement/string_refinement.cpp

```diff
@@ -326,85 +469,126 @@ std::size_t expr_cast<std::size_t>(const exprt& val_expr)
 /// same value as the next index that is present in the index set.
 /// We do so by traversing the array backward, remembering the
 /// last value that has been initialized.
-void string_refinementt::concretize_string(const exprt &expr)
+static void concretize_string(
```

Comment:
`get` and `max_string_length` can be made `const`.

Reply:
Arguments have been made const.